adapters/rds/db_cluster.go

```diff
@@ -368,3 +369,23 @@ func NewDBClusterSource(client rdsClient, accountID string, region string) *sour
 		OutputMapper: dBClusterOutputMapper,
 	}
 }
+
+func DBClusterMetadata() sdp.AdapterMetadata {
+	return sdp.AdapterMetadata{
+		Type:            "rds-db-cluster",
+		DescriptiveName: "RDS Cluster",
+		SupportedQueryMethods: &sdp.AdapterSupportedQueryMethods{
+			Get:               true,
+			List:              true,
+			Search:            true,
+			GetDescription:    "Get a parameter group by name",
+			ListDescription:   "List all RDS parameter groups",
+			SearchDescription: "Search for a parameter group by ARN",
+		},
+		TerraformMappings: []*sdp.TerraformMapping{
+			{TerraformQueryMap: "aws_rds_cluster_parameter_group.arn"},
```

Comment:
```suggestion
			{TerraformQueryMap: "aws_rds_cluster.cluster_identifier"},
```

Seems to be a copy paste error